src/main/kotlin/no/nav/familie/tilbake/service/dokumentbestilling/felles/pdf/JournalføringTjeneste.kt

```diff
@@ -0,0 +1,77 @@
+package no.nav.familie.tilbake.service.dokumentbestilling.felles.pdf
+
+import no.nav.familie.kontrakter.felles.dokarkiv.ArkiverDokumentRequest
+import no.nav.familie.kontrakter.felles.dokarkiv.AvsenderMottaker
+import no.nav.familie.kontrakter.felles.dokarkiv.Dokument
+import no.nav.familie.kontrakter.felles.dokarkiv.FilType
+import no.nav.familie.kontrakter.felles.dokarkiv.IdType
+import no.nav.familie.tilbake.behandling.domain.Behandling
+import no.nav.familie.tilbake.behandling.domain.Fagsak
+import no.nav.familie.tilbake.behandling.domain.Verge
+import no.nav.familie.tilbake.integration.familie.IntegrasjonerClient
+import no.nav.familie.tilbake.service.dokumentbestilling.felles.Adresseinfo
+import no.nav.familie.tilbake.service.dokumentbestilling.felles.BrevMetadata
+import no.nav.familie.tilbake.service.dokumentbestilling.felles.BrevMottaker
+import no.nav.familie.tilbake.service.dokumentbestilling.fritekstbrev.JournalpostIdOgDokumentId
+import org.slf4j.LoggerFactory
+import org.springframework.stereotype.Service
+
+@Service
+class JournalføringTjeneste(private val integrasjonerClient: IntegrasjonerClient) {
```

Comment:
Jeg tror familie område bruker "Service" på slutten for service klasser. Er det noen spesiell årsak til å bruke "Tjeneste" på slutten?

Reply:
Nei. Jeg hadde tenkt å ha en commit til slutt med navneendringer, slik at det passet med vår kode, men jeg glemte det.